src/Common/src/CoreLib/System/Type.cs

```diff
@@ -23,7 +23,11 @@ public abstract partial class Type : MemberInfo, IReflect
         public abstract string FullName { get; }
 
         public abstract Assembly Assembly { get; }
+#if MONO
+        public override abstract Module Module { get; }
```

Comment:
Keep the comment from the refsrc code:

```suggestion
        // Workaround for JIT bug with bad VTable lookup
        //
        //      MemberInfo mi = typeof (System.DBNull);
        //      System.Console.WriteLine (mi.Module);
        //
        public override abstract Module Module { get; }
```

I assume you checked if this bug still exists?

Reply:
I doesn't have any failures related to removing this workaround. I left it only due to potential API change. If that API change is OK then I can revert it.

Reply:
It should be fine to remove the workaround.

Reply:
Done